lib/manageiq/api/client/mixins/custom_inspect_mixin.rb

```diff
@@ -0,0 +1,30 @@
+module CustomInspectMixin
+  extend ActiveSupport::Concern
+
+  DEFAULT_CUSTOM_INSPECT_EXCLUSIONS = [:@custom_inspect_exclusions].freeze
```

Comment:
None of this juggling of instance variables needs to happen if the definition is done at the class level.